webpack/scenes/Subscriptions/Manifest/CdnConfigurationTab/UpstreamServerTypeForm.js

```diff
@@ -220,6 +220,7 @@ const UpstreamServerTypeForm = ({
 
         <ActionGroup>
           <Button
+            ouiaId="upstream-configuration-update-button"
```

Comment:
```suggestion
            ouiaId="network-sync-configuration-update-button"
```

Reply:
@jeremylenz Thanks for the update.
I will wait until https://github.com/Katello/katello/pull/10050 is merged.